xcm/xcm-executor/src/traits/token_matching.rs

```diff
@@ -48,8 +48,9 @@ impl<Instance> MatchesNonFungible<Instance> for Tuple {
 }
 
 /// Errors associated with [`MatchesFungibles`] operation.
+#[derive(Debug, PartialEq, Eq)]
 pub enum Error {
-	/// Asset not found.
+	/// The given asset is not handled. (According to [`XcmError::AssetNotFound`])
```

Comment:
We might want to clarify the intention behind the `AssetNotFound` error variant -- we do use it as a sort of mechanism to indicate that a `MatchesFungibles` implementation doesn't handle the asset, in which case, we should also rename the variant to `AssetNotHandled`.

Reply:
agree, renamed `AssetNotFound->AssetNotHandled` for match errors,

everywhere were comments like:
```
        // Check we handle this asset.
		let (asset_id, amount) = Matcher::matches_fungibles(what)?;

or 

        // Check we handle this asset.
		let amount: Currency::Balance =
			Matcher::matches_fungible(what).ok_or(Error::AssetNotHandled)?;
```


anyway, in one of my first version of this PR, when I changed doc here, I also renamed `AssetNotFound->AssetNotHandled`, but then I reverted it not to make so much noise :)